bluesky/run_engine.py

```diff
@@ -111,7 +111,7 @@ class RunEngine:
 
     _loop = loop  # just a convenient way to inspect the global event loop
     state = LoggingPropertyMachine(RunEngineStateMachine)
-    _UNCACHEABLE_COMMANDS = ['pause', 'subscribe', 'unsubscribe']
+    _UNCACHEABLE_COMMANDS = ['pause', 'subscribe', 'unsubscribe', 'stage']
```

Comment:
Redundant unstaging does no harm, so it doesn't *have* to go in this list. Let's defer the discussion of whether it should to a better solution (one that addressed Case 2 above) in a separate PR.